docs/Developers/Debug_Bears.rst

```diff
@@ -0,0 +1,52 @@
+Debug Bears
+===========
+
+This document provide a brief overview of coala's Debugging Interface.
+Debugging Interface will help user to debug the Bear code and move through
+it using ``pdb`` Interface
+
+There are two way to invoke the coala's Debgger
```

Comment:
Rather make a header with "Usage", instead of this sentence.
+
Typo in Debgger (please be careful with typos, you make them quite often, I don't want to fix typos all the time)